### ISSUE TITLE
Is the open source code wrong?

### ISSUE THREAD

**Issue by zhiweihu1103** [hwwang55/PathCon#6]
When I want to view the complete PathCon model code, I found that there are some missing codes? The following are something puzzle me. 
* Point one: Regarding the experiment in Figure 6 in the paper, it cannot be found in the provided code about the **Embedding+Mean** and  **Embedding+Attention**. 
* Point two: The formula 15 in the paper uses softmax, but the code provided uses sigmoid?
* Point three: The formulas 13 and 14 in the paper have no traces of use in the embedding mode in the provided code, but the paper mentions that PathCon is more effective after introducing formulas 13 and 14. Can you explain it here?
The above are some of my questions, I look forward to your reply!!!

**Comment by hwwang55** [comment]
Hi there, thanks for your interest in our work! Regarding your questions:
1 & 3. I checked the repository. Embedding+Mean is implemented, but Embedding+Attention is not included. I guess this is due to the version issue. I will fix this soon. Actually this is just some small modifications on lines 78, 79 in model.py. BTW, using attention can only slightly improve the performance (see figure 6) for both "embedding" and "rnn", so relax and feel comfortable to use "mean" :)
2. We indeed use softmax. See line 217 in model.py. I guess what you mean is line 86 in model.py, where self.scores_normalized = tf.sigmoid(self.scores). Here self.scores_normalized is not used for training but for the purpose of evaluation. See the comment in line 85 for details.

**Comment by zhiweihu1103** [comment]
> Hi there, thanks for your interest in our work! Regarding your questions:
> 1 & 3. I checked the repository. Embedding+Mean is implemented, but Embedding+Attention is not included. I guess this is due to the version issue. I will fix this soon. Actually this is just some small modifications on lines 78, 79 in model.py. BTW, using attention can only slightly improve the performance (see figure 6) for both "embedding" and "rnn", so relax and feel comfortable to use "mean" :)
> 2. We indeed use softmax. See line 217 in model.py. I guess what you mean is line 86 in model.py, where self.scores_normalized = tf.sigmoid(self.scores). Here self.scores_normalized is not used for training but for the purpose of evaluation. See the comment in line 85 for details.

Thank you for your reply. Nice.
